test/integration/targets/vmware_guest/tasks/clone_set_manual_mac_address.yml

```diff
@@ -0,0 +1,43 @@
+- name: Power off clone source VMs
+  vmware_guest:
+    validate_certs: False
+    hostname: "{{ vcenter_hostname }}"
+    username: "{{ vcenter_username }}"
+    password: "{{ vcenter_password }}"
+    name: "{{ item.name }}"
+    datacenter: "{{ dc1 }}"
+    folder: "{{ item.folder }}"
+    state: poweredoff
+  loop: "{{ virtual_machines }}"
+
+- name: clone VM from template and set manual mac address
+  vmware_guest:
+    validate_certs: False
+    hostname: "{{ vcenter_hostname }}"
+    username: "{{ vcenter_username }}"
+    password: "{{ vcenter_password }}"
+    name: "{{ 'mac_manual_' + item.name }}"
```

Comment:
Please call the VM `test_vm1`.

See: https://docs.ansible.com/ansible/devel/dev_guide/platforms/vmware_guidelines.html#vm-names-should-be-predictable

Reply:
@goneri Are you content with the test case as it is in the aforementioned rebased version? https://github.com/ansible/ansible/pull/64511/files#diff-e69056a42b1d753a1664f4a2747d23a1

Reply:
Yes, if we ignore the error message, the result looks. Thank you.